README.md

```diff
@@ -1,12 +1,149 @@
+	static #cloudlyDB: storage.Database<Layout>
+	static get cloudlyDB(): storage.Database<Layout> {
+		return (Storage.#cloudlyDB =
+			Storage.#cloudlyDB ??
+			storage.Database.create<Layout>(
+				{
+					silos: { user: { type: "collection", idLength: 8, retainChanged: true } },
+				},
+				Storage.environment.archive,
+				Storage.environment.BufferBackend
+			) ??
+			gracely.server.misconfigured("databaseBuffer", "Missing environment variables to open database."))
+	}
+	static initialize(environment: Record<string, any>) {
+		this.environment = environment
+	}
+}
+```
+here I use the dependency `gracely` to generate errors. Note that the `Storage.environment.archive` & `Storage.environment.BufferBackend` needs to correspond to the bindings defined in wrangler.toml. The initialize function needs to be called with the environment by the worker before utilizing the database.
+
+Now the database can be used by the worker as follows:
+```
+import { Storage } from "./Storage"
+import { User } from "../model/User"
+
+export async function store(user: User, organization: string): Promise<User> {
+	return await Storage.cloudlyDB.user.partition(organization).store(user)
+}
+```
+
+## API
+Once the database has been created following the example in the section `Integration` the following functions can be applied on the database. The Document type can be found in `Database/Document.ts`
+
+### store
+Stores document(s) in the database.
+```
+store(document: User & Partial<Document>): Promise<(User & Document) | undefined>
+store(documents: (User & Partial<Document>)[]): Promise<((User & Document) | undefined)[]>
+```
+### load
+Loads document(s) form the database by either specifying the id, ids or a query in the form specified in the type `Database/Selection/index.ts`. A locus can be returned if the query includes more documents than the specified limit or 1000 as default. The locus can be used to continue the listing from where the previous one stopped.
+
+```
+load(id: Identifier): Promise<(T & Document) | undefined>
+load(ids: Identifier[]): Promise<((Document & T) | undefined)[] & { locus?: string }>
+load(selection?: Selection): Promise<(Document & T)[] & { locus?: string }>
+```
+### remove
+The remove function will delete the document and return a boolean representing the success of the operation.
+```
+remove(id: Identifier): Promise<boolean>
+remove(id: Identifier[]): Promise<boolean[]>
+```
+### partition
+The partition function can be used to create partitions, theoretically infinitly many partitions can be created but don't go crazy with it.
+```
+partition(...partition: Identifier[]): S 
+```
+## Indecies
+There exists 3 types of keys in the Database, `id`, `changed` and `doc`; each adds functionality to the database.
+The definitions below are based on how they look in the `Archive`; the buffer uses a small variation of these keys where the type of the document as well as the slash following the type is omitted.
```

Comment:
Instead of `small variation` i suggest one of these:
- `smaller part`
- `subset`
- `fragment`